src/core/test/fuzzing/src/test/scala/FuzzingTest.scala

```diff
@@ -112,6 +114,7 @@ class FuzzingTest extends TestBase {
     val exemptions: Set[String] = Set(
       "com.microsoft.ml.spark.TrainClassifier",
       "com.microsoft.ml.spark.TrainRegressor",
+      "com.microsoft.ml.spark.TuneHyperparameters",
```

Comment:
this one too

Reply:
for the python tests I need to add a corresponding parameter space python file first